custom_components/venta/sensor.py

```diff
@@ -140,6 +169,16 @@ class VentaSensorEntityDescription(
         is not None,
         value_func=lambda coordinator: coordinator.data.measure.get("WaterLevel"),
     ),
+    VentaSensorEntityDescription(
```

Comment:
"Dust" is shown as PM2.5 in the Venta-App

Reply:
![7988406B-C186-49D9-86CB-5E7C154405F5_1_102_o](https://github.com/Michsior14/ha-venta/assets/1864448/df65d984-f256-427d-a780-13ff58bc483d)

Reply:
```
{
   "Header":{
      "DeviceType":2,
      "MacAdress":"f8...:9c",
      "Hash":"...",
      "Error":0,
      "DeviceName":""
   },
   "Action":{
      "Power":true,
      "FanSpeed":2,
      "TargetHum":55,
      "Timer":0,
      "Boost":false,
      "SleepMode":false,
      "ChildLock":false,
      "Automatic":true,
      "SysLanguage":3,
      "CleanLanguage":1,
      "TempUnit":0,
      "DisplayLeft":0,
      "DisplayRight":1,
      "Reset":0,
      "ConINet":0,
      "DelUser":false
   },
   "Info":{
      "SWDisplay":"V2.009",
      "SWPower":"V1.001",
      "SWTouch":"V1.3.0",
      "SWWIFI":"V19.4.8",
      "CleanMode":false,
      "RelState":[
         false,
         false,
         false,
         false
      ],
      "TimerT":0,
      "OperationT":147640,
      "DiscIonT":26401,
      "CleaningT":52801,
      "FilterT":19596,
      "UVCOnT":0,
      "UVCOffT":0,
      "CleaningR":0,
      "Warnings":97
   },
   "Measure":{
      "Temperature":24.4,
      "Humidity":39.1,
      "Dust":1,
      "WaterLevel":2,
      "FanRpm":0
   }
}
```